jstz_cli/src/account/mod.rs

```diff
@@ -59,7 +74,7 @@ fn delete_account(alias: String, cfg: &mut Config) -> Result<()> {
     let mut input = String::new();
     io::stdin().read_line(&mut input)?;
 
-    if !["y", "Y", "yes", "Yes"].contains(&input.trim()) {
+    if !input.trim().eq(&alias) {
```

Comment:
Naïve question: Why did the previous version require `.trim()` and this version doesn't?

Reply:
No sure if I understand - both versions have trim. It is so that in case that there is an accidental whitespace in the beginning/end it removes it for better usability.

Reply:
Oh yes, I misread the diff. My bad